src/components/notebook/macro-animal-row.tsx

```diff
@@ -10,7 +10,7 @@ import {
 
 import "./macro-panel.scss";
 
-const kMaxCritters = 60;
+const kMaxCritters = 72; // 20% more than the max possible abundance
```

Comment:
Perhaps a constant would make sense here, e.g.
```
  const kLowSunAbundanceFactor = 1.2;
  const kMaxCritters = Math.round(60 * kLowSunAbundanceFactor);
```
where the `Math.round()` is to prevent any floating point weirdness, e.g. 71.999...,

Reply:
Yeah, I think that is clearer.

Reply:
There was one other opportunity to use the constant down below.